src/main/scala/elasticbeanstalk/elasticbeanstalk.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2012-2015 Pellucid Analytics
+ * Copyright 2015 Daniel W. H. James
+ * Copyright 2015 Andrea Lattuada
```

Comment:
FYI, Pellucid Analytics is no longer associated with this project. Work up until 2015 was under their copyright, but as the current maintainer, work since then has been under mine.